src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/hmpps/HmppsAuditEvent.kt

```diff
@@ -4,6 +4,7 @@ import java.time.ZoneId
 import java.time.ZonedDateTime
 import java.time.format.DateTimeFormatter
 
+@Suppress("PropertyName")
```

Comment:
What's this for?

Reply:
kolint and IntelliJ has validation on property name to start with lower case. Audit event class require a property called "when", however, this is a keyword in kotlin, therefore we need to wrap the name "when" with back ticks, and it will break the name validation, therefore I have to suppress that for this class

Reply:
Cool